lib/ansible/modules/files/archive.py

```diff
@@ -423,7 +423,8 @@ def main():
                 try:
                     if format == 'zip':
                         arcfile = zipfile.ZipFile(dest, 'w', zipfile.ZIP_DEFLATED, True)
```

Comment:
probably not related to this PR but if someone will be doing future refactoring, please replace this with `with` block using a context manager.